src/main/java/com/faforever/client/replay/ReplayService.java

```diff
@@ -111,6 +125,82 @@
   private final FafService fafService;
   private final ModService modService;
   private final MapService mapService;
+  private final EventBus eventBus;
```

Comment:
We try to not use EventBus any longer but use ApplicationPublisher from Spring for new changes / features etc.

Reply:
@Brutus5000 said the opposite in Zulip. Which one is it? :)

Reply:
EventBus is deprecated

Reply:
If he said the opposite he got it the wrong way arround

Reply:
I misremembered; @Brutus5000 said to use Spring. @Geosearchef and @norraxx, however, seemed to prefer EventBus. I really don't mind either way, just want to be sure we're all on the same page.
I can't find anything about `EventBus` being deprecated. Where did you hear about that?

Reply:
Sorry i meant unstable/Beta 
![image](https://user-images.githubusercontent.com/19996713/70454027-10646980-1aaa-11ea-969f-cd0661756779.png)

Reply:
I think it was even marked with unstable in older guava versions, also we wanna stay with Spring. In the beginning tho nobody knew Spring had an Event System as well.

Reply:
I've replaced `EventBus` with Spring methods for this PR.
Since Spring's event system works a bit differently, I had to remove the `@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)` from the ReplayVaultController, because otherwise, Spring would instantiate a new controller whenever the Event was triggered.